packages/expo-prevent-screen-capture/ios/EXPreventScreenCapture/EXPreventScreenCaptureModule.m

```diff
@@ -0,0 +1,72 @@
+// Copyright 2018-present 650 Industries. All rights reserved.
+
+#import <EXPreventScreenCapture/EXPreventScreenCaptureModule.h>
+
+@interface EXPreventScreenCaptureModule ()
+
+@property (nonatomic, weak) UMModuleRegistry *moduleRegistry;
+@property (nonatomic, strong) UIView *blockView;
+
+@end
+
+@implementation EXPreventScreenCaptureModule
+
+UM_EXPORT_MODULE(ExpoPreventScreenCapture);
+
+- (id)init {
+    if ( self = [super init] ) {
+        self.blockView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, [[UIScreen mainScreen] bounds].size.width, [[UIScreen mainScreen] bounds].size.height)];
+        [self.blockView setBackgroundColor:[UIColor whiteColor]];
```

Comment:
Does it block the screen like in nothing is shown on the screen whatsoever?

Reply:
correct, it's just a plain white view (although the red statusbar at the top that indicates you're recording still shows). here are videos of this in action on android and ios (I changed iOS to show a black screen rather than a white screen when recording, to match Android):
ios: 
![RPReplay-Final1590089144](https://user-images.githubusercontent.com/35579283/82598029-f4e0ad00-9b77-11ea-96f2-3fe241baf866.gif)

android: 
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/35579283/82598287-6c164100-9b78-11ea-9992-59b779653d5c.gif)

Reply:
Oh, interesting, I've never seen an app do that, that's why it sounded strange to me. If that's the same across the platforms that sounds well!